packages/component-model-manuscript/entities/manuscript/index.js

```diff
@@ -319,10 +319,18 @@ class Manuscript extends BaseModel {
   // would mean applying the input to the manuscript first
   static removeOptionalBlankReviewers(input) {
     const itemIsBlank = item => item.name + item.email === ''
-    const filteredReviewers = input.suggestedReviewers.filter(
+    const filteredSuggestedReviewers = input.suggestedReviewers.filter(
```

Comment:
Would it not make more sense to make sure that blank or null entries are not sent from the client rather than removing them on the server?

Reply:
Ideally we should do both. Given this is impacting users on a seemingly daily basis, it makes sense to do it on the server side first. Happy to do the client side in separate PR.

Reply:
Agreed

Reply:
ok ill mark this as resolved then